index.html

```diff
@@ -98,7 +97,7 @@ <h2 class="work-title">Tonic</h2>
           <article class="work" id="work-img-2">
             <img
               class="work-img-2"
-              src="assets/bg-mobile-2.svg"
+              src="assets/work2.svg"
               alt="First Project"
             />
             <div class="item2">
```

Comment:
- [ ] As requested by the previous reviewer please make sure our project cards follow the figma template your design :point_down:  the buttons sre not in the right place
![image](https://user-images.githubusercontent.com/97521557/206544648-23913ee7-c7b4-4bae-931b-493b8993881e.png)
 
figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206544822-8e2b369c-1248-4cda-a310-eedf08cc111b.png)